Nodejs/Product/Nodejs/Project/NodeModulesNode.cs

```diff
@@ -362,9 +377,9 @@ private class NpmPathProvider : INpmPathProvider {
             }
         }
 
-#endregion
+        #endregion
 
-#region Updating module hierarchy
+        #region Updating module hierarchy
 
         internal void ReloadHierarchySafe() {
```

Comment:
Thanks, this sounds like a better idea and should cut down the size of the change considerably.